src/components/standalone/security/ips/IpsEnableRuleModal.vue

```diff
@@ -0,0 +1,85 @@
+<script lang="ts" setup>
+import { useI18n } from 'vue-i18n'
+import type { Rule } from '@/components/standalone/security/ips/IpsDisabledRules.vue'
+import { ref, watchEffect } from 'vue'
+import { ubusCall } from '@/lib/standalone/ubus'
+import { getAxiosErrorMessage, NeInlineNotification, NeModal } from '@nethesis/vue-components'
+
+const { t } = useI18n()
+
+const { rule } = defineProps<{
+  rule?: Rule
+}>()
+
+const _rule = ref<Rule | undefined>()
+watchEffect(() => {
+  if (rule != undefined) {
+    _rule.value = rule
+  }
+})
+
+const emit = defineEmits<{
+  close: []
+  enabled: []
+}>()
+
+const loading = ref(false)
+const error = ref<Error>()
+
+function handleClose() {
+  if (!loading.value) {
+    emit('close')
+  }
+}
+
+function enableRule() {
+  loading.value = true
+  error.value = undefined
+  ubusCall('ns.snort', 'enable-rule', {
+    gid: _rule.value!.gid,
+    sid: _rule.value!.sid
+  })
+    .then(() => {
+      emit('enabled')
+    })
+    .catch((reason: Error) => {
+      error.value = reason
+    })
+    .finally(() => {
+      loading.value = false
+    })
+}
+</script>
+
+<template>
+  <NeModal
+    :cancel-label="t('common.cancel')"
+    :close-aria-label="t('common.close')"
+    :primary-button-disabled="loading"
+    :primary-button-loading="loading"
+    :primary-label="t('common.delete')"
+    :title="t('standalone.ips.enable_rule_modal_title')"
+    :visible="rule != undefined"
+    kind="warning"
```

Comment:
Not sure this should be a warning modal :thinking: I can't find it in the mockups :cry:

Reply:
Technically it should be, in theory you are enabling a rule that potentially locks the traffic for a considerable amount of users, not sure this should be an `info`

Reply:
Ok then, let's keep it as a warning